CHANGES.md

```diff
@@ -13,6 +13,7 @@ Unreleased changes refer to our current [master branch](https://github.com/openc
 * Send event invitation mails to several attendees, mixing internal and external recipients
 * Fix folder hierarchy synchronization issues on mailbox subfolders
 * Old mails are now synchronized after account cleanup
+* Better support to attachments in OpenChange client against OpenChange server
```

Comment:
Can you please improve the CHANGELOG entry? I don't know what has been improved with this line only.